### ISSUE TITLE
Build fails on case-sensitive filesystems

### ISSUE THREAD

**Issue by thehappycoder** [zevarito/Licode-ErizoClientIOS#54]
clang: error: no such file or directory: '/Volumes/ExternalSSD/projects/3rd/Licode-ErizoClientIOS/ErizoClient/rtc/RTCIceCandidate+JSON.m'

Indeed, such file does not exist. But RTCICECandidate+JSON.m exists.

```
Antons-iMac:Licode-ErizoClientIOS anton_kuzmin$ grep -R "RTCIceCandidate+JSON.m" .
./ErizoClientIOS.xcodeproj/project.pbxproj:		78D593711B7C17FF00EDF432 /* RTCIceCandidate+JSON.m in Sources */ = {isa = PBXBuildFile; fileRef = 78D5935F1B7C17FF00EDF432 /* RTCIceCandidate+JSON.m */; };
./ErizoClientIOS.xcodeproj/project.pbxproj:		78D5935F1B7C17FF00EDF432 /* RTCIceCandidate+JSON.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = "RTCIceCandidate+JSON.m"; sourceTree = "<group>"; };
./ErizoClientIOS.xcodeproj/project.pbxproj:				78D5935F1B7C17FF00EDF432 /* RTCIceCandidate+JSON.m */,
./ErizoClientIOS.xcodeproj/project.pbxproj:				78D593711B7C17FF00EDF432 /* RTCIceCandidate+JSON.m in Sources */,
Binary file ./ErizoClientIOS.xcworkspace/xcuserdata/anton_kuzmin.xcuserdatad/UserInterfaceState.xcuserstate matches
```

Xcode 8.3.2

ExternalSSD volume is formatted as case-sensitive mac os extended drive

**Comment by zevarito** [comment]
Thanks for reporting, if you already have a patch for this don't hesitate on submit a PR.

**Comment by zevarito** [comment]
Should be fixed in 0.5.0 release, there is branch if you want to try.
